### PR TITLE
Circular Linked List implemented python

### PR DESCRIPTION
**Pull request by Shishir198**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Implemented circular linked list with append,prepend,remove,insert and printAll with proper Documentation

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
YES

### CODE REVIEW

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Shishir198 please do following modifications
* For friendly user interface, take input from the user & display appropriate message after each operation
* Create a count function for counting the total number of nodes present at that time.
* Create a exit function, for exiting out from the program

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @Shishir198 please incorporate the changes requested by @siddharth25pandey along with  these
1. Add sample input and output below the code section in commented form. 
2. Add one line comment before each function telling what it is doing

**Comment by Shishir198** [comment]
ThankYou Mentors for pointing out the mistakes and needs.
I have updated and pushed my changes according to your change request.
Kindly check through it.

File: Python/ds/Circular_LinkedList.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+    # Setting First Value as HEAD
+    l  = circular_llist(Node(user_input[0]))
+
+    # Appending values in list
+    for inputs in range(1,len(user_input)):
+        l.create(Node(user_input[inputs]))
+
+    l.printAll()
+    l.append(Node(10))
```
Comment (Kajol-Kumari) [inline Python/ds/Circular_LinkedList.py]: Hey @Shishir198 please take these inputs from the user i.e the value to be appended at the start, value to be appended at the end etc.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Please incorporate the suggested changes, rest looks good to me.

File: Python/ds/Circular_LinkedList.py (hunk 1 above)
Comment (Shishir198) [reply Python/ds/Circular_LinkedList.py]: Okay mam. Do you mean for every function I had to ask user for input.

Comment (Kajol-Kumari) [reply Python/ds/Circular_LinkedList.py]: Yes @Shishir198, please ask user for the input for each operation instead of hardcoding it.

Comment (Shishir198) [reply Python/ds/Circular_LinkedList.py]: @Kajol-Kumari Mam! I have done the changes your requested.

**Review by siddharth25pandey — APPROVED** [review]
@Shishir198 Remove the Merge Conflicts

**Comment by Shishir198** [comment]
@siddharth25pandey Sir why I am always facing merge conflicts in python/README.md .Am I doing something wrong.

**Comment by siddharth25pandey** [comment]
> @siddharth25pandey Sir why I am always facing merge conflicts in python/README.md .Am I doing something wrong.

@Shishir198 Did some changes.
Add the **README file again** with proper details

**Comment by Shishir198** [comment]
@siddharth25pandey I have updated the Readme. Hope this time it will not create any issue

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :)

**Review by SKAUL05 — APPROVED** [review]

**Review by siddharth25pandey — APPROVED** [review]
@Shishir198 Looks fine to me
Able to merge
Thanks!

**Comment by iamrajiv** [comment]
@all-contributors please add @Shishir198 for code

**Pull request merged**